plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/core-profiler.spec.js

```diff
@@ -464,18 +467,20 @@ test.describe( 'Store owner can skip the core profiler', () => {
 		await test.step( 'Confirm that the store is in coming soon mode after skipping the core profiler', async () => {
 			await page.goto( 'wp-admin/admin.php?page=wc-admin' );
 			await expect(
-				page.getByRole( 'menuitem', { name: 'Store coming soon' } )
+				page
+					.getByRole( 'menuitem' )
+					.filter( { hasText: 'coming soon' } )
 			).toBeVisible();
 		} );
 	} );
 
 	test( 'Can connect to WooCommerce.com', async ( { page } ) => {
-		await test.step( 'Go to WC Home and make sure the total sales is visible', async () => {
-			await page.goto( 'wp-admin/admin.php?page=wc-admin' );
-			await page
-				.getByRole( 'menuitem', { name: 'Total sales' } )
-				.waitFor( { state: 'visible' } );
-		} );
+		// await test.step( 'Go to WC Home and make sure the total sales is visible', async () => {
+		// 	await page.goto( 'wp-admin/admin.php?page=wc-admin' );
+		// 	await page
+		// 		.getByRole( 'menuitem', { name: 'Total sales' } )
+		// 		.waitFor( { state: 'visible' } );
+		// } );
```

Comment:
"Total Sales" is missing on WooCommerce home on Pressable, that's why I commented it until I figure out why is this. 

<img width="736" alt="Screenshot 2024-08-26 at 11 48 06" src="https://github.com/user-attachments/assets/bd81f2dc-532b-4a95-b9da-d74dc168cd84">

Reply:
Done in [e38abeb](https://github.com/woocommerce/woocommerce/pull/50927/commits/e38abebdd21f79dc9363c300eaa07a9a8b432eb0)

Reply:
@adimoldovan, I reverted previous commit in [36e1d7d](https://github.com/woocommerce/woocommerce/pull/50927/commits/36e1d7da816f521a8d5f4014f3bfbadc840a39d4) and re-enabled "Total sales" check in [153c369](https://github.com/woocommerce/woocommerce/pull/50927/commits/153c369e599b37a0461dc6d83920f3c422a18763). 

I figured out why it wasn't displayed in the first place, fixed that on the Pressable website directly, and now the test is passing.